content/concepts/connections.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Connections
+weight: 3
+---
+
+A libp2p connection allows two peers to read and write data to each other. 
+Peers can connect through transport protocols. Transport protocols are core 
+abstractions of libp2p and offer extensibility. As a modular networking stack, 
+libp2p is transport-agnostic and does not enforce the implementation of a specific 
+transport protocol. Though the support of modern transport protocols is a primary 
+focus for libp2p, implementers may choose to support multiple types of transport. 
+Learn more about transport protocols in libp2p on the 
+[transport guide]((/concepts/transports/)). 
+
+Let's first outline key terminology to help us understand connections in libp2p.
+
+## Core libp2p components
+
+- **Peer**: a participant in a libp2p network.
+- **Peer-to-Peer (P2P)**: a distributed network in which workloads are 
+  shared between *Peers*.
```

Comment:
Does this belong in the connections documentation or in a general glossary?

Also referencing https://docs.ipfs.io/concepts/glossary/ just in case you haven't seen it yet.